shells/backends/gcp_tf_backend/src/driver.py

```
@@ -178,11 +178,16 @@ def _get_and_validate_clp(self, clp_details, gcp_backend_resource: str, logger)
         clpr_res_fam = clp_details_resource.ResourceFamilyName
         if (clpr_res_fam != 'Cloud Provider' and clpr_res_fam != 'CS_CloudProvider') or \
                 clp_res_model not in GCP_MODELS:
-            logger.error(f"Cloud Provider does not have the expected type: {clpr_res_fam}")
-            raise ValueError(f"Cloud Provider does not have the expected type:{clpr_res_fam}")
+            logger.exception(f"Cloud Provider does not have the expected type:{clpr_res_fam}")
```

There is no "raise" in this case. You only write to log. Dont we need to raise an error here?